src/routes/tabs.js

```diff
@@ -1,4 +1,6 @@
-import React, { PureComponent } from 'react';
+/* eslint react/no-multi-comp:0 */
```

Comment:
I believe these changes will need to be made to our `tabNotification.js` file as well.

Reply:
do we need to? the top of the tab notification says the following:
```
Created a hacky TabNavigator with initial screen of Notifications. This way user can navigate to related notification.
```

Reply:
I meant more the `DrawerWrapper` content but if that isn't applicable. that's fine :+1: